apps/backend/src/deposits/cash-deposit.service.ts

```diff
@@ -53,12 +53,12 @@ export class CashDepositService {
     const {
       program,
       date,
-      location: { location_id }
+      location: { pt_location_id }
```

Comment:
where is `pt_location_id` calculated? 

when the input is given as just `location_id` in any event, how does this work?